plugins/ci/pkg/models/models.go

```diff
@@ -150,13 +155,43 @@ func maybeAddSlash(input string) string {
 }
 
 // SetDefaults sets configuration defaults
-func (c *Configuration) SetDefaults() {
+func (c *Configuration) SetMountedPathDefaults(basePath, repoBasePath string) error {
+	logrus.Infof("using SetMountedPathDefaults(%s)", basePath)
+	c.RepoBasePath = repoBasePath
+	c.Options.TempFolder = filepath.Join(basePath, "tmp/insightsTemp")
+	err := os.MkdirAll(c.Options.TempFolder, os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("SetMountedPathDefaults: %v", err)
+	}
+	c.Options.TempFolder = maybeAddSlash(c.Options.TempFolder)
+
+	// TODO: images are copied via script insights-ci.sh, what now?
+	c.Images.FolderName = filepath.Join(basePath, "tmp/insightsTempImages")
+	err = os.MkdirAll(c.Images.FolderName, os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("SetMountedPathDefaults: %v", err)
+	}
+	c.Images.FolderName = maybeAddSlash(c.Images.FolderName)
+
+	return nil
+}
+
+// SetDefaults sets configuration defaults
+func (c *Configuration) SetPathDefaults() {
```

Comment:
We can change it later but to avoid many changes and possible regression errors I kept the normal ctx using old folder structures